HPC/README.md

```diff
@@ -36,39 +36,65 @@ ArmPL: v23.04 & later (included in the ACfL compiler)
 MPI: Open MPI v4.1.4 & later (the latest official release)
 
 ## Instructions for setting up the HPC cluster for best performance
-We recommend using [AWS ParallelCluster](https://docs.aws.amazon.com/parallelcluster/latest/ug/what-is-aws-parallelcluster.html) (previously known as [CfnCluster](http://cfncluster.readthedocs.io)) to deploy and manage HPC clusters  on AWS EC2. AWS ParallelCluster 3.5.1 is a tool that can automatically set up the required compute resources, job scheduler, and shared filesystem commonly needed to run HPC applications. This section covers step-by-step instructions on how to set up or upgrade the tools and software packages to the recommended versions on a new ParallelCluster. Please refer to the individual sub-sections if you need to update certain software package on an existing cluster. For a new cluster setup, you can use [this template](scripts/hpc7g-ubuntu2004-useast1.yaml) to create a Ubuntu 20.04 cluster and run the [setup script](scripts/install-tools-headnode-ubuntu2004.sh) on the headNode to install the required tools and packages.
+We recommend using [AWS ParallelCluster](https://docs.aws.amazon.com/parallelcluster/latest/ug/what-is-aws-parallelcluster.html) (previously known as [CfnCluster](http://cfncluster.readthedocs.io)) to deploy and manage HPC clusters  on AWS EC2. AWS ParallelCluster 3.5.1 is a tool that can automatically set up the required compute resources, job scheduler, and shared filesystem commonly needed to run HPC applications. This section covers step-by-step instructions on how to set up or upgrade the tools and software packages to the recommended versions on a new ParallelCluster. Please refer to the individual sub-sections if you need to update certain software package on an existing cluster. For a new cluster setup, you can use [this template](scripts-setup/hpc7g-ubuntu2004-useast1.yaml) and replace the subnet and ssh key information from your account to create a Ubuntu 20.04 cluster. The command to create a new cluster is
+```
+pcluster create-cluster --cluster-name test-cluster --cluster-configuration hpc7g-ubuntu2004-useast1.yaml
+```
+The cluster creation process takes about 10 minutes. You can find headNode information under the EC2 console page once the creation process is finished.
+```
+HeadNode | i-01489594da7c76f77 | Running | c7g.4xlarge	2/2 checks passed	| No alarms | us-east-1b	ec2-3-227-12-112.compute-1.amazonaws.com	3.227.12.112	––	disabled	| pcluster-aarch64-ubuntu2004-useast1-HeadNodeSecurityGroup-xxxxxxxx	xxxxxxx	2023/05/09 09:17 GMT-5
```

Comment:
For describing what to find on the EC2 console, please provide a screenshot of what they should expect to see.  

Is there a way to get this information off the command line as well?